modules/sandboxed-containers-deleting-operator-web-console.adoc

```diff
@@ -15,7 +15,7 @@ You can delete the {sandboxed-containers-operator} by deleting the catalog subsc
 
 .Procedure
 
-. Navigate to the *Operators* → *OperatorHub* page.
-. Search for `{sandboxed-containers-first}` and then select the Operator.
-. Click *Uninstall*.
-. Delete the `openshift-sandboxed-containers-operator` namespace.
+. From the *Administrator* perspective, navigate to the *Operators* → *Installed Operators* page.
+. Search for `{sandboxed-containers-operator}` using the *Search by name* field.
```

Comment:
Search for the {sandboxed-containers-operator} using the *Search by name* field

I added "the" and removed the back ticks.